### PR TITLE
ISP co-badge full abi list

### PR DESCRIPTION
**Pull request by gammam**
Added all ABI list managed by ISP co-badge service.
The following ABI are not in abi.json file

06325
05026
03186
06340* 
06260*
03309
02002

### CODE REVIEW

**Review by fabriziofff — CHANGES_REQUESTED** [review]
@gammam in order to pass the CI checks we need:
- Add the missing abi to the `abi.json` file
- The name should be the same

```
[01010] with name "BANCO DI NAPOLI S.P.A.O" should be "BANCO DI NAPOLI S.P.A."
abi 02002 cannot have an empty name
abi 03309 cannot have an empty name
abi 06260 cannot have an empty name
[06340] with name "" should be "CASSA DI RISPARMIO DEL FRIULI VENEZIA GIULIA"
abi 03186 cannot have an empty name
abi 05026 cannot have an empty name
abi 06325 cannot have an empty name
```

File: status/cobadgeServices.json [hunk 1]
```diff
@@ -1120,7 +1120,156 @@
...
+        "abi": "06165",
+        "name": "CASSA DI RISPARMIO DI FOLIGNO"
+      },
+      {
+        "abi": "06225",
+        "name": "CASSA DI RISPARMIO DEL VENETO"
+      },
+      {
+        "abi": "06260",
+        "name": ""
```
Comment (thisisjp) [inline status/cobadgeServices.json:1214]:
```suggestion
        "name": "Cassa di Risparmio di Pistoia e della Lucchesia"
```

File: status/cobadgeServices.json [hunk 2]
```diff
@@ -1120,7 +1120,156 @@
...
+        "abi": "03060",
+        "name": "BANCA BRIGNONE SPA"
+      },
+      {
+        "abi": "05526",
+        "name": "BANCO SAN GIORGIO"
+      },
+      {
+        "abi": "06325",
+        "name": ""
```
Comment (thisisjp) [inline status/cobadgeServices.json:1270]:
```suggestion
        "name": "Cassa di Risparmio di Tortona S.p.A."
```

File: status/cobadgeServices.json [hunk 3]
```diff
@@ -1120,7 +1120,156 @@
...
+        "abi": "06930",
+        "name": "BANCA MONTE PARMA SPA"
+      },
+      {
+        "abi": "03186",
+        "name": ""
+      },
+      {
+        "abi": "05026",
+        "name": ""
```
Comment (thisisjp) [inline status/cobadgeServices.json:1250]:
```suggestion
        "name": "BPU Banca - Banche Popolari Unite"
```

File: status/cobadgeServices.json [hunk 4]
```diff
@@ -1120,7 +1120,156 @@
...
+        "abi": "06385",
+        "name": "CASSA DI RISPARMIO IN BOLOGNA SPA"
+      },
+      {
+        "abi": "06930",
+        "name": "BANCA MONTE PARMA SPA"
+      },
+      {
+        "abi": "03186",
+        "name": ""
```
Comment (thisisjp) [inline status/cobadgeServices.json:1246]:
```suggestion
        "name": "B@nca 24-7 S.p.a."
```

File: status/cobadgeServices.json [hunk 5]
```diff
@@ -1120,7 +1120,156 @@
...
+        "abi": "03240",
+        "name": "BANCA DI TRENTO E BOLZANO"
+      },
+      {
+        "abi": "03296",
+        "name": "BANCA FIDEURAM - S.P.A."
+      },
+      {
+        "abi": "03309",
+        "name": ""
```
Comment (thisisjp) [inline status/cobadgeServices.json:1166]:
```suggestion
        "name": "Banca Infrastrutture Innovazione e Sviluppo S.p.A."
```

File: status/cobadgeServices.json [hunk 6]
```diff
@@ -1120,7 +1120,156 @@
...
+        "abi": "01010",
+        "name": "BANCO DI NAPOLI S.P.A"
+      },
+      {
+        "abi": "01025",
+        "name": "SANPAOLO IMI S.P.A."
+      },
+      {
+        "abi": "02002",
+        "name": ""
```
Comment (thisisjp) [inline status/cobadgeServices.json:1134]:
```suggestion
        "name": "Banca Commerciale Italiana"
```

**Review by thisisjp — APPROVED** [review]
I've added the missing bank names

File: status/cobadgeServices.json [hunk 7]
```diff
@@ -1120,6 +1120,154 @@
       {
         "abi": "06225",
         "name": "CASSA DI RISPARMIO DEL VENETO"
+      },
+      {
+        "abi": "01010",
+        "name": "BANCO DI NAPOLI S.P.A"
```
Comment (fabriziofff) [inline status/cobadgeServices.json:1126]:
```suggestion
        "name": "BANCO DI NAPOLI S.P.A."
```

**Review by fabriziofff — CHANGES_REQUESTED** [review]

**Review by fabriziofff — APPROVED** [review]

**Pull request merged**